File: Prefix Sum/2848.Points That Intersect With Cars/Solution.py
class Solution:
    def numberOfPoints(self, nums: List[List[int]]) -> int:
        q = [0] * 110
        for u, v in nums:
            q[u] += 1
            q[v + 1] -= 1
        ans = s = 0
        for v in q:
            s += v
            if s:
                ans += 1
        return ans

File: Prefix Sum/2848.Points That Intersect With Cars/Solution.java
class Solution {
    public int numberOfPoints(List<List<Integer>> nums) {
        int[] q = new int[110];
        for (List<Integer> e : nums) {
            q[e.get(0)]++;
            q[e.get(1) + 1]--;
        }
        int ans = 0, s = 0;
        for (int v : q) {
            s += v;
            if (s > 0) {
                ans++;
            }
        }
        return ans;
    }
}